example/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "typeorm-sql-js-example",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "ts-node src/index.ts",
+    "build": "tsc",
+    "typeorm": "typeorm-ts-node-commonjs"
+  },
+  "dependencies": {
+    "typeorm": "0.3.22",
+    "sql.js": "^1.8.0",
```

Comment:
`sql.js@1.13.0` and `typescript@5.8.3` are newer.

The file could be sorted with `npx sort-package-json`.